### ISSUE TITLE
Leverage on Rstudio's "Import data" functionality to introduce "Reading the data into R" in "Starting with data"

### ISSUE THREAD

**Issue by DaniMori** [datacarpentry/R-ecology-lesson#700]
My suggestion:

Before introducing the function `read_csv()` and get the students hands-on code with reading data, let them use the "Import dataset" UI functionality in Rstudio.

The rationale:

- Many learners will also lack a proper [mental model](https://carpentries.github.io/instructor-training/02-practice-learning/#cognitive-development-and-mental-models) of the file path hierarchy, which is necessary to integrate such new concepts as "relative vs. absolute path" or file extensions. The "Import dataset" tool avoids the need to learn and apply those concepts in order to make a quick start with the reading data functionality. Also, the code automatically generated by the "Import dataset" tool can be used as a starting point to introduce those concepts, if necessary.

- One of the hardest problems I have (incidentally) observed in starting writing syntax is in understanding how every single character matters: case sensitivity, quotation marks, commas, non-printable characters, parentheses, etc. may pose a singular challenge to newcomers. The "Import dataset" tool offers the possibility to make a start with an already meaningful syntax and play around to fine-tune it for the learner's purpose with much less [*extraneous* cognitive load](https://carpentries.github.io/instructor-training/05-memory/#why-guided-practice-is-important) than writing the whole code from scratch.

- As widely explained in lesson 0: "Before we start", Rstudio offers a suite of tools to power up R's capabilities. I think that the "Import dataset" interface is one such tool that may help ease the learning of R syntax for importing data by providing the learners with a familiar graphical interface, on the one hand, and at the same time giving them the opportunity to explore and play around with the R syntax it automatically creates on the other.

**Comment by tobyhodges** [comment]
Thanks @DaniMori for suggesting this change. The lesson underwent a major update and reorganisation when https://github.com/datacarpentry/R-ecology-lesson/pull/887 was merged. As this issue relates to content in a version of the lesson before that update took place, I will close it. Please open a new issue if you believe that some or all of the changes you proposed here remain relevant to the redesigned lesson, linking to this thread where appropriate.
